### PR TITLE
[4.2] Convert mod_articles_latest to new structure

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
This PR converts mod_articles_latest  to new structure

### Testing Instructions
1. Use Joomla 4.2
2. Create an instance of Articles - Latest module. Note how it works.
3. Apply patch
4. Delete administrator/cache/autoload_psr4.php
5. Refresh the page, make sure the module still displays the same as before.

### CODE REVIEW

**Comment by richard67** [comment]
@joomdonation Could you fix the PHP CS issue reported by drone?

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 5baff2a85ec7b1c66baf6de81a40ced5ae3106dd<br /><br />Tested successfully in Joomla 4.2.0-beta1-dev of 1 June.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37947">issues.joomla.org/tracker/joomla-cms/37947</a>.</sub>

**Comment by pabloarias** [comment]
I have tested this item :white_check_mark: successfully on 5baff2a85ec7b1c66baf6de81a40ced5ae3106dd<br /><br />Hello:

I used Joomla! version 4.2.0-beta1-dev and PHP 8.1.5.

Installed Blog Sample Data and created module Articles - Latest. It works fine.

If I delete the file before apply the path, it works fine.

After applying the path, it works too.

Thank you and best regards<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37947">issues.joomla.org/tracker/joomla-cms/37947</a>.</sub>

**Comment by richard67** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37947">issues.joomla.org/tracker/joomla-cms/37947</a>.</sub>

File: modules/mod_articles_latest/src/Helper/ArticlesLatestHelper.php [hunk 1]
```diff
@@ -146,4 +151,21 @@ public static function getList(Registry $params, ArticlesModel $model)
 
 		return $items;
 	}
+
+	/**
+	 * Retrieve a list of article
```
Comment (richard67) [inline modules/mod_articles_latest/src/Helper/ArticlesLatestHelper.php]:
```suggestion
	 * Retrieve a list of articles
```

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**